src/Microsoft.Diagnostics.EventFlow.Inputs.EventSource/EventDataExtensions.cs

```diff
@@ -29,6 +30,11 @@ public static EventData ToEventData(this EventWrittenEventArgs eventSourceEvent)
             eventPayload["EventId"] = eventSourceEvent.EventId;
             eventPayload["EventName"] = eventSourceEvent.EventName;
             eventPayload["ActivityID"] = ActivityPathString(eventSourceEvent.ActivityId);
+            var eventContext = ContextResolver.GetRequestContext<object>();
+            if (eventContext != null)
+            {
+                eventPayload["EventContext"] = eventContext;
```

Comment:
1. Is "RequestContext" a better name?
2. What if I want to set the request context at the time when the event is created (not at the time when it is processed by the input)?